sonarr4kinstall.sh

```diff
@@ -63,16 +72,28 @@ fi
 
 echo_progress_start "Generating configuration"
 
+# Start sonarr to config
+systemctl stop sonarr.service >>$log 2>&1
+
 cat > /home/${user}/.config/sonarr4k/config.xml << EOSC
 <Config>
+  <LogLevel>info</LogLevel>
+  <UpdateMechanism>BuiltIn</UpdateMechanism>
+  <Branch>develop</Branch>
+  <BindAddress>127.0.0.1</BindAddress>
   <Port>8882</Port>
   <SslPort>9898</SslPort>
-  <UrlBase>/sonarr4k</UrlBase>
-  <BindAddress>127.0.0.1</BindAddress>
-  <UpdateMechanism>none</UpdateMechanism>
+  <EnableSsl>False</EnableSsl>
+  <LaunchBrowser>False</LaunchBrowser>
+  <AuthenticationMethod>None</AuthenticationMethod>
+  <UrlBase>sonarr4k</UrlBase>
+  <UpdateAutomatically>False</UpdateAutomatically>
```

Comment:
For posterity: valid 

https://github.com/Sonarr/Sonarr/blob/0a2b109a3fe101e260b623d0768240ef8b7a47ae/src/NzbDrone.Core/Configuration/ConfigFileProvider.cs#L206

Reply:
Prior XML line `  <UpdateMechanism>none</UpdateMechanism>` was invalid causing sonarr to error out on launch. 

added `  <UpdateMechanism>BuiltIn</UpdateMechanism>` with` <UpdateAutomatically>False</UpdateAutomatically> ` to ensure only the 1st master instance can update sonarr for future safeproofing.